docs/source/Plugin/P014.rst

```diff
@@ -24,6 +24,44 @@ Used libraries: |P014_usedlibraries|
 Supported hardware
 ------------------
 
+**SI7013:**
+===========
+
+Has support for humidity, temperature and ADC values 
+
+* ± 3% RH (max), 0–80% RH  (0 to 100% RH operating range)
+* ±0.4 °C (max), –10 to 85 °C (–40 to +125 °C operating range)
+* Operating Voltage (1.9 to 3.6 V)
+* Low Power Consumption: 150 μA active current ; 60 nA standby current
+* Analog To Digital convertor builtin.

```

Comment:
converter (typo)

Reply:
??? was showing ok in my browser.
![image](https://user-images.githubusercontent.com/1547916/182049898-fd22e90c-45af-46ca-b9c0-f969eb8d4d4e.png)

Reply:
oh, I see, misspelling.

Reply:
done